src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/IconUrlToImageCacheConverter.cs

```diff
@@ -80,6 +81,7 @@ public object Convert(object[] values, Type targetType, object parameter, Cultur
             if (IsEmbeddedIconUri(iconUrl))
             {
                 var iconEntry = Uri.UnescapeDataString(iconUrl.Fragment).Substring(1); // skip the '#' in a URI fragment
+                var iconEntryNormalized = PathUtility.StripLeadingDirectorySeparators(iconEntry) ;
```

Comment:
is iconEntry used anywhere now?

Reply:
Only in the line below. I left the variable to increase readability and minimize changeset, since extracting the icon entry involves multiple steps/operation calls.

Reply:
Analyzing the MSIL code, it seems to have less operations if the intermediate variable is removed. So, I will remove `ìconEntry`

Short version:

```asm
// string packageFile = PathUtility.StripLeadingDirectorySeparators(Uri.UnescapeDataString(iconUrl.Fragment).Substring(1));
IL_0130: ldfld class [System]System.Uri NuGet.PackageManagement.UI.IconUrlToImageCacheConverter/'<>c__DisplayClass5_0'::iconUrl
IL_0135: callvirt instance string [System]System.Uri::get_Fragment()
IL_013a: call string [System]System.Uri::UnescapeDataString(string)
IL_013f: ldc.i4.1
IL_0140: callvirt instance string [mscorlib]System.String::Substring(int32)
IL_0145: call string [NuGet.Common]NuGet.Common.PathUtility::StripLeadingDirectorySeparators(string)
IL_014a: stloc.s 15
```

Readable version:

```asm
// string filename = Uri.UnescapeDataString(iconUrl.Fragment).Substring(1);
IL_0130: ldfld class [System]System.Uri NuGet.PackageManagement.UI.IconUrlToImageCacheConverter/'<>c__DisplayClass5_0'::iconUrl
IL_0135: callvirt instance string [System]System.Uri::get_Fragment()
IL_013a: call string [System]System.Uri::UnescapeDataString(string)
IL_013f: ldc.i4.1
IL_0140: callvirt instance string [mscorlib]System.String::Substring(int32)
IL_0145: stloc.s 15
// string packageFile = PathUtility.StripLeadingDirectorySeparators(filename);
IL_0147: ldloc.s 15
IL_0149: call string [NuGet.Common]NuGet.Common.PathUtility::StripLeadingDirectorySeparators(string)
IL_014e: stloc.s 16
```